tests/deps/jester-#head/jester/private/utils.nim

```diff
@@ -1,7 +1,6 @@
 # Copyright (C) 2012 Dominik Picheta
 # MIT License - Look at license.txt for details.
-import parseutils, strtabs, strutils, tables, net, mimetypes, asyncdispatch, os
-from cgi import decodeUrl
+import parseutils, strtabs, strutils, tables, net, mimetypes, asyncdispatch, os, uri
```

Comment:
Errr, wtf? Why is Jester in Nim's source tree?

Reply:
Because so that tests are self-contained without Nimble dependencies.

Reply:
You can install a specific commit hash of Jester using Nimble, why not do that? or use a git submodule?

Reply:
We used to do that but there was a period in the transition to 0.19 where Nimble would just fail. It also made local testing harder, I don't run these travis/appveyor scripts on my local machines.

Reply:
So don't put the Nimble commands into the travis script :/